receiver/prometheusreceiver/config.go

```diff
@@ -54,6 +54,11 @@ type Config struct {
 	// in incorrect rate calculations.
 	UseStartTimeMetric   bool   `mapstructure:"use_start_time_metric"`
 	StartTimeMetricRegex string `mapstructure:"start_time_metric_regex"`
+	// DisableStartTime disables start time calculation of all metrics, which significantly reduces the memory usage of the receiver.
+	// Start timestamp is strongly recommended for Sum, Histogram, and ExponentialHistogram points, removing the start timestamp
+	// may limit the ability of other components and backends to offer full functionality with these metrics.
+	// Use only if you know what you are doing.
+	DisableStartTime bool `mapstructure:"disable_start_time"`
```

Comment:
Let's move it above UseStartTimeMetric as it's a gate above that. So if !DisableStartTime then UseStartTimeMetric doesn't do anything, right?

Reply:
make sense. updated.